From 41d67a9e2cff81532723d6a2f79bfa857f582596 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Wed, 30 May 2018 15:36:58 +0800
Subject: [PATCH] arm64: dts: rockchip: add rockchip,adc-grps for RK3308 EVB

By default, we enable all of ADC groups on RK3308 EVB.

Change-Id: Id89adab1c7d9cde22fb4ac3d5dcd3b18a8a96da3
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi | 1 +
 arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index cf8a61b0945c..d49fb07547ec 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -239,6 +239,7 @@
 &acodec {
 	status = "okay";
 
+	rockchip,adc-grps = <0 1 2 3>;
 	rockchip,loopback-grp = <3>;
 	hp-ctl-gpios = <&gpio0 RK_PA1 GPIO_ACTIVE_HIGH>;
 	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
index 0c8d9f3ab7d6..3c8f3eee3c8f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
@@ -245,6 +245,7 @@
 &acodec {
 	status = "okay";
 
+	rockchip,adc-grps = <0 1 2 3>;
 	rockchip,loopback-grp = <0>;
 	hp-ctl-gpios = <&gpio0 RK_PA1 GPIO_ACTIVE_HIGH>;
 	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
-- 
2.35.3

